libs/evcs-service/src/main/java/uk/gov/di/ipv/core/library/evcs/service/EvcsService.java

```diff
@@ -207,7 +208,7 @@ public List<EvcsGetUserVCDto> getUserVCs(
         return evcsClient.getUserVcs(userId, evcsAccessToken, List.of(states)).vcs();
     }
 
-    private void persistUserVCs(
+    private HttpResponse<String> persistUserVCs(
```

Comment:
@undermad @thebauSoftwire I left a comment on https://github.com/govuk-one-login/ipv-core-back/pull/3157/files#r2089533732 as I'm not sure this is going to work for what we need here